fix: multiple typos

Hamza-ye opened the pull request:
Fixes of multiple typos I found while reading, it actually still hard to navigate and fix collectively the typos found individually while reading. I hope my contribution be of help :)

Hamza-ye committed:
Fix multiple typos

Fixes of multiple typos I found while reading, it actually still hard to navigate and fix collectively the typos found individually while reading. I hope my contribution be of help :)

Philip-Larsen-Donnelly commented on src/commonmark/en/content/developer/web-api.md:
```diff
@@ -10499,7 +10499,7 @@ The analytics resource lets you specify a range of query parameters:
 <td>userOrgUnit</td>
 <td>No</td>
-<td>Explicitly define the user org units to utilize, overrides organisation units associated with current user, multiple identifiers can be separated by semi-colon.</td>
+<td>Explicitly define the user org units to utilize, overrides organisation units associated with the current user, multiple identifiers can be separted by semicolon.</td>
```
typo: separted

Philip-Larsen-Donnelly commented on src/commonmark/en/content/developer/web-api.md:
```diff
@@ -10475,13 +10475,13 @@ The analytics resource lets you specify a range of query parameters:
 <td>inputIdScheme</td>
 <td>No</td>
-<td>Identifier scheme to use for metadata items in the query request, can be identifier, code or attributes.</td>
+<td>Identifier scheme to use for metadata items in the query request, can be an identifier, code or attributes.</td>
```
To me this reads slightly wrongly with the additional "an" - because it cannot be _any_ identifier, it must be literally "UID".

Hamza-ye replied:
Yes, thanks, and I actually corrected as I read, sometimes things my slips without noticing them, thanks for the correction.

Philip-Larsen-Donnelly commented on src/commonmark/en/content/developer/web-api.md:
```diff
@@ -10603,8 +10603,8 @@ will filter the analytics response. You can use it like this:
 of a ready-made table, you can provide the *tableLayout* parameter with
-true as value. Instead of generating a plain, normalized data source,
-the analytics resource will now generate the data in table layout. You
+true as a value. Instead of generating a plain, normalized data source,
```
I think it's better with either "true as value" or "true as the value", but not "true as a value".

Philip-Larsen-Donnelly commented on src/commonmark/en/content/developer/web-api.md:
```diff
@@ -11320,7 +11320,7 @@ periods explicitly in the
       &dimension=ou:O6uvpzGd5pu;fdc6uOvgoji
 
-The *filter* parameter can be used to filter a response without
+The *filter* parameter can be used to filter response without
```
I think this should be "to filter a response"

Philip-Larsen-Donnelly commented on src/commonmark/en/content/developer/web-api.md:
```diff
@@ -12260,7 +12260,7 @@ event data. A request looks like this:
       &bbox=-13.2682125,7.3721619,-10.4261178,9.904012&includeClusterPoints=false
 
-The cluster response provides the count of underlying points, the centre
+The cluster response provides the count of underlying points, the center
```
the documentation contains both UK and US variations of English, so either of these are OK.

Philip-Larsen-Donnelly commented on src/commonmark/en/content/developer/web-api.md:
```diff
@@ -13490,7 +13490,7 @@ DHIS2. Geospatial features are stored together with organisation units.
 the organisation unit dimension for the analytics resource. It is
 recommended to read up on the analytics api resource before continuing
-reading this section. You must use the GET request type, and only JSON
+tp read this section. You must use the GET request type, and only JSON
```
typo: tp -> to

Hamza-ye replied:
thanks again and sorry for these, I really see dhis2 an elegant solution that solved a lot of our problems, and as I read the documentation sometimes when I find only one error and when I go to fix it I can't help it but look for other things that may be okay and I may make them worse so thanks for your clarifications and effort

Philip-Larsen-Donnelly reviewed (changes requested):
There are a few minor comments, but otherwise a lot of nice fixes! Thanks!

Philip-Larsen-Donnelly commented:
Hi @Hamza-ye 

I think you fixed a lot of stuff in this, so would like to keep it open and just edit the few comments I made! ;)

Hamza-ye committed:
Fix the issues you pointed out

thanks for your comments, here are the fixes for the issues you pointed out in my previous commit.
